hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableHandler.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.hadoop.ozone.recon.tasks;
+
+import org.apache.commons.lang3.tuple.Triple;
+import org.apache.hadoop.hdds.utils.db.Table;
+import org.apache.hadoop.hdds.utils.db.TableIterator;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.HashMap;
+
+/**
+ * Interface for handling PUT, DELETE and UPDATE events for size-related
+ * tables for OM Insights.
+ */
+public interface OmTableHandler {
+
+  /**
+   * Handles a PUT event for size-related tables by updating both the data
+   * sizes and their corresponding record counts in the tables.
+   *
+   * @param event                    The PUT event to be processed.
+   * @param tableName                Table name associated with the event.
+   * @param sizeRelatedTables        Tables Requiring Size Calculation.
+   * @param objectCountMap           A map storing object counts.
+   * @param unreplicatedSizeCountMap A map storing unReplicated size counts.
+   * @param replicatedSizeCountMap   A map storing replicated size counts.
+   * @throws IOException If an I/O error occurs during processing.
+   */
+  void handlePutEvent(OMDBUpdateEvent<String, Object> event,
```

Comment:
Pls check if these methods needs to declare throws IOException. I don't see usage. Pls check

Reply:
Thanks for pointing it out I have removed and logged it!